drift/lib/src/runtime/types/sql_types.dart

```diff
@@ -168,6 +168,10 @@ class BlobType extends SqlType<Uint8List> {
 
   @override
   Uint8List? mapFromDatabaseResponse(dynamic response) {
+    if (response is String) {
```

Comment:
Could you also add a test verifying this behavior [here](https://github.com/simolus3/moor/blob/0a2cc40ba5bb0aa78c6192c479ddd1b22bef8ad3/drift/test/types/blob_test.dart#L6)? Just test that `mapFromDatabaseResponse` with a string returns an expected list of bytes.

Reply:
Sure, will do so later this evening

Reply:
Done, test added